src/app/components/project-summary/project-summary.component.html

```diff
@@ -0,0 +1,72 @@
+<div>
+  <div class="row">
+    <div class="LHS-details column">
+      <div class="project-details">
+        <span>
+          <strong>Project Name: </strong> {{ currentProject?.name }}
+        </span>
+      </div>
+
+      <div class="project-details">
+        <span
+          ><strong>Start Date: </strong>
+          {{
+            this.resourceService.prettyPrintDate(
+              currentProject?.start_date?.toLocaleString("en-GB", {
```

Comment:
This all calculation should be in ts and if we are using in multiple places we can move this in utils folder